infer/src/base/DB.ml

```diff
@@ -133,6 +136,13 @@ module Results_dir = struct
       Utils.create_dir (path_to_filename (Abs_source_dir source) []) )
 
 
+  let clean_specs_dir () =
+    (* create dir just in case it doesn't exist to avoid errors *)
+    Utils.create_dir specs_dir ;
+    Array.iter (Sys.readdir specs_dir) ~f:(fun specs ->
+        Filename.concat specs_dir specs |> Sys.remove )
+
+        
```

Comment:
Is this a rebase error or is the specs/ directory really back?

Reply:
> Is this a rebase error or is the specs/ directory really back?

Yes. I have updated to comply with latest infer